source/content/guides/local-development/02-configuration.md

```diff
@@ -214,13 +214,7 @@ This file is ignored by the `.gitignore` file in [WordPress](https://github.com/
 
 Pantheon sites that install WordPress 5.5 include a `wp-config-local-sample.php` file. Older sites can copy the [wp-config-local-sample.php](https://github.com/pantheon-systems/WordPress/blob/default/wp-config-local-sample.php) file on GitHub to the same directory as the site's `wp-config.php`, or create one in that location as shown here.
 
-The following can be used as a starting point for `wp-config-local.php`. Replace the database values with the values from your local environment, and the key/salt values with your unique phrase (generated from [WordPress.org](https://api.wordpress.org/secret-key/1.1/salt/)).
-
-<Accordion title={"Full text of wp-config-local-sample.php"} id={"full-wp-config-local-sample"}>
-
-GITHUB-EMBED https://github.com/pantheon-systems/WordPress/blob/default/wp-config-local-sample.php php GITHUB-EMBED
```

Comment:
@rachelwhitton we should probably retain the link to this file, right?

Reply:
It's linked above on line 215

Reply:
actually i made a mistake on this one i didn't mean to delete line 217 i meant to edit it, will fix now